client/src/lib/browser.js

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from "react";
+
+export default function Browser() {
```

Comment:
Can you rename this to `useAvailableHeight` and export it by name (not default), so we can later add other hooks or functionality related to the browser.